x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.test.tsx

```diff
@@ -46,18 +46,19 @@ jest.mock('react-router-dom', () => {
   };
 });
 
-jest.mock('../../../../../common/endpoint/service/host_isolation/utils', () => {
-  return {
-    isIsolationSupported: jest.fn().mockReturnValue(true),
-  };
-});
+// FIXME:PT Fix tests after refactor
```

Comment:
I wouldn't worry about fixing these tests as this entire side panel is going to be removed in `8.16` (in `8.15` we enabled the expandable flyout by default, and users can only revert back to the old flyout with a feature flag...

Reply:
ahhh. Good to know. Ok. I'll just remove the commented out code and leave it as is (I actually think it working fine now and I just forgot to remove the comment)